client/src/app/shared/misc/screen.service.ts

```diff
@@ -10,7 +10,12 @@ export class ScreenService {
     this.refreshWindowInnerWidth()
   }
 
-  isInSmallView () {
+  isInSmallView (marginLeft = 0) {
+    if (marginLeft > 0) {
+      const contentWidth = this.getWindowInnerWidth() - marginLeft
+      return contentWidth < 800
+    }
+
     return this.getWindowInnerWidth() < 800
   }
```

Comment:
Since this modifies the logic of a service function, have you checked instances of components using it for regressions?

Reply:
Yes,  every call is without any parameter, if `marginLeft = 0`  it will be the same behaviour than before because the default value of `marginLeft` is `0`.